notebooks/official/experiments/comparing_pipeline_runs.ipynb

```diff
@@ -66,6 +66,15 @@
     {
```

Comment:
<p>remove future tense: will</p><p> </p><p>eg.,</p><p>Next, you build the pipeline .....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>done</p>